.ci/run.sh

```diff
@@ -13,6 +13,12 @@ pushd "${tests_repo_dir}"
 .ci/run.sh
 testcidir=$(dirname "$0")
 
+# docker version may need to be upgraded to let make proto go on arm64
+arch=$(go env GOARCH)
+if [ "$arch" == "arm64" ]; then
+	"../agent/.ci/run_arm64.sh"
```

Comment:
This path looks odd - shouldn't it be something like the following?:

```bash
cidir=$(dirname "$0")
"${cidir}/run_arm64.sh"
```

Reply:
but it won't work as the current dir is .../tests. run_arm64.sh is not found in the path of ".ci/run_arm64.sh"

Reply:
Ack! I was only looking at that line in isolation ;)